testing/run_test.sh

```diff
@@ -9,6 +9,14 @@ if [ $HOST == 'Linux' ]; then
     FLAGS=':Z'
 fi
 
+if [ "x$E2E_REPO" != "x" ]; then
+    if [ $(stat -c %a $HOME/.kube/config) != "660" ]; then
+        # kubeconfig needs to be readable inside the iqe container for the oc command to work.
+        chmod 660 $HOME/.kube/config
+    fi
+    E2E_MOUNT="-v $HOME/.kube:/iqe_venv/.kube${FLAGS} -v $E2E_REPO:/e2e-deploy${FLAGS} -w /e2e-deploy"
+fi
+
```

Comment:
Please let me know if this shouldn't be here. This lets the load_test_customer_data script use the IQE container using our existing scripts instead of me writing a new script solely for that purpose. I was on the fence about whether to do it one way versus the other, but it seemed like if this change wasn't too invasive, that this script's purpose could be expanded a little.